core/arch/arm/kernel/kern.ld.S

```diff
@@ -134,11 +134,21 @@ SECTIONS
 #include <rodata_unpaged.ld.S>
 #else
 		*(.rodata .rodata.*)
+#ifndef CFG_CORE_ASLR
+		. = ALIGN(8);
+		KEEP(*(SORT(.scattered_array*)));
+#endif
 #endif
 		. = ALIGN(8);
 		__rodata_end = .;
 	}
 
+#if !defined(CFG_WITH_PAGER) && defined(CFG_CORE_ASLR)
```

Comment:
Why can't this always be used even when ASLR isn't enabled?

Reply:
It can, but doesn't have to. I thought having a section called `.data.rel.ro` could be slightly misleading when ASLR is disabled (and therefore the executable is not relocatable).
I can simplify if you prefer.

Reply:
Makes sense, let's keep it as it is now.